alica_tests/autogenerated/src/Behaviour/NotToTrigger.cpp

```diff
@@ -18,7 +18,6 @@ NotToTrigger::NotToTrigger()
     // Add additional options here
     this->callCounter = 0;
     this->initCounter = 0;
-    setTrigger(alicaTests::TestWorldModel::getOne()->trigger1.get());
```

Comment:
test_alica_behaviourtrigger will now fail. Since it uses alicaTests::TestWorldModel::getOne()->trigger1->run(true); And you removed the dependency of ITrigger. Now it is cleaner but you will have to modify the trigger testcases to call doTrigger() instead of ITrigger->run()

Reply:
yes, that should be done already in https://github.com/rapyuta-robotics/alica/pull/146